### PR TITLE
 Fix: Added Reflect.ownKeys polyfill to support IE11

### PR DESCRIPTION
**Pull request by SebastianSmolorz**
Issue: #314

Tested in IE11, to reproduce try:

```Person.query().with('nationality').get()```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "@babel/plugin-proposal-class-properties": "^7.4.0",
     "@babel/plugin-transform-runtime": "^7.4.3",
     "@babel/preset-env": "^7.4.3",
+    "@types/core-js": "^2.5.0",
```
Comment (kiaking) [inline package.json]: Seems like it adds only type definition? I think we need actual core-js package as well?

**Comment by kiaking** [comment]
Thanks! But the conflict looks like its on builded files. Could you try removing the files under `dist` folder from the commits?

**Comment by SebastianSmolorz** [comment]
Oh dear I think I rebased before pulling so it duplicated the commits...is there any way to revert this without force pushing?

**Comment by kiaking** [comment]
@SebastianSmolorz Ah, Not so sure. But it is OK to force push this PR though.

**Pull request merged**